package.json

```diff
@@ -20,6 +20,7 @@
     "webpack-dev-server": "^4.13.3"
   },
   "scripts": {
-    "start": "webpack serve --mode development"
+    "start": "webpack serve --mode development",
+    "build": "webpack"
```

Comment:
- [ ] [OPTIONAL] Please use the flag `--open` instead of `--mode development` for running the webpack dev server on your browser

Reply:
well resolved! ✔️